docs/FAQ/FAQ-LightningNetwork.md

```diff
@@ -200,6 +201,10 @@ export BTCPAYGEN_ADDITIONAL_FRAGMENTS="opt-save-storage"`
 
 Other pruning options are [documented here](https://github.com/btcpayserver/btcpayserver-docker/blob/master/README.md#generated-docker-compose)
 
+### Can I use my existing LN node with BTCPay?
+
+If you already have a well connected lightning node with sufficient inbound liquidity, you may want to use it with BTCPay instead of the included lightning node. While it is technically possible to do so, the BTCPay documentation does not include all information for custom deployments, including this scenario. Any contributor who would like to include this information should make a PR to this repository for review.
```

Comment:
```suggestion
If you already have a well connected lightning node with sufficient inbound liquidity, you may want to use it with BTCPay instead of the included lightning node. 
To do so, go to the lightning node settings page of your store (Store > Settings > Lightning nodes > Modify). 
On this page you can edit the connection string to integrate an external node.
The connection strings vary per lightning implementation, the settings page contains the documentation for the connection settings to give you further guidance.
```

Reply:
Thanks @dennisreimann !

Might also be useful to add something of the sorts:

`If you have trouble finding the connection strings for the external LN node you wish to integrate, please contact the software or service provider for assistance`

So that we don't get too much support time helping people that can't find the string coz of cluncky third-party services that might hide that info deeeeeeeeeeeep out of sight.

Reply:
Unfortunately that won't work as the connection string details are specific to BTCPay Server. That's why they are documented on the LN setup page.

Reply:
Oh okay.